sys/mips/mips/stack_machdep.c

```diff
@@ -41,6 +41,14 @@ __FBSDID("$FreeBSD$");
 #include <machine/pcb.h>
 #include <machine/regnum.h>
 
+static bool
+stack_addr_ok(struct thread *td, u_register_t sp, u_register_t stack_pos)
+{
+	u_register_t va = sp + stack_pos;
+
+	return (kstack_contains(td, va, sizeof(u_register_t)));
```

Comment:
```suggestion

	return (kstack_contains(td, sp + stack_pos, sizeof(u_register_t)));
```

Maybe inline this variable?

Reply:
Maybe I should just inline this entire function.  Originally I didn't have kstack_contains so this function was slightly larger, but maybe it's redundant now.

Reply:
I inlined the function.